src/applications/facility-locator/components/ServiceTypeAhead.jsx

```diff
@@ -90,7 +78,7 @@ class ServiceTypeAhead extends Component {
           selectedItem,
         }) => (
           <div>
-            <label {...getLabelProps()}>
+            <label {...getLabelProps()} htmlFor="service-type-ahead-input">
```

Comment:
This actually does fix an issue. The value of the label's  `htmlFor` didn't match that of the `input`'s. This fixes that.

![image](https://user-images.githubusercontent.com/1915775/60116888-faa2f800-9746-11e9-865e-1cce460df613.png)

Reply:
Great catch Nick! This fixes a violation I hadn't previously seen but just verified. Gives me some ideas about the typeahead too.